Development-Guide/CAN-Filter-Guide/README.adoc

```diff
@@ -0,0 +1,68 @@
+= CAN Message Filter Guide
+
+== Introduction
+This is a basic guide on how to make a CAN filter.
+The two main components of a filter are an ID and a Mask.
+The ID will be the unique identifier for the message you are looking for, and the Mask will determine which bits of the ID are important to sift through (this makes the proccess much more efficient and flexible). 
```

Comment:
The ID *is* the unique identifier

the Mask *determines* which bits of the ID

process typo